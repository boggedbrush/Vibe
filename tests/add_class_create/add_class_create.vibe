# VibeSpec: 1.0
patch_type: add_class
file: hello.py
--- code: |
class Fareweller:
    def say_bye(self, name):
        print(f"Byeâ€‘bye, {name}!")
