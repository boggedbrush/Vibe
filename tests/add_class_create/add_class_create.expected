# hello.py
def greet(name):
    print(f"Hello, {name}!")

class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')

class Fareweller:
    def say_bye(self, name):
        print(f"Byeâ€‘bye, {name}!")
